src/api-client.ts

```diff
@@ -0,0 +1,114 @@
+import * as httpm from "@actions/http-client";
+
```

Comment:
What do you think about calling this file `gh-api-client` to make it more explicit about what it actually is?